### ISSUE TITLE
Atom Launcher and Atom Panel crashing / unable to be enabled

### ISSUE THREAD

**Issue by Magentium-Chris** [ozonos/bug-tracker-BETA#13]
Finding recently that Atom Launcher and Atom Panel are both showing as "Error Loading Extension" under Gnome Tweak. Restarting my Laptop fixed the issue some times, but since the Launcher crashed, NO ICONS show up when clicking on "All Applications.

Please let me know what information you require

**Comment by sergioad** [comment]
Similar issue here with Atom launcher

**Comment by Magentium-Chris** [comment]
Just had it happen again >.< What kind of information would be useful for this?

**Comment by Magentium-Chris** [comment]
Temp fix that I've so far found is to press ALT + F2 , type R and hit Enter. This resets GNOME but not really a viable fix >.<

**Comment by sergioad** [comment]
yes, with ALT + F2 + R + enter it is temporally fixed but not permanentlly, this should be fixed as soon as possible, I am not on OzonOS in this moment majorlly because of this issue :smile:

**Comment by cldx** [comment]
I will post introductions of what i need to track down the bugs later on, can't do it right now since i'm at work. Chances are good though it's already fixed in master cause this reminds me of an older bug report.

**Comment by Magentium-Chris** [comment]
Thanks cldx! Loving the OS so far!

**Comment by cldx** [comment]
Allright, if you find the time do the following:

Disable all extensions except ONE that is making problems.
Run this in a Terminal : journalctl /usr/bin/gnome-session -f -o cat

Try to recreate the problem (by enabling/disabling the Extensions, clicking or whatever made it happen for you).

Check the Terminal for any relevant output, if there is some, paste it here :)

Repeat for any other Extension that makes problems. You can also run this with all Extensions enabled but the log might blow up with messages in no particular order (f.e. there's a bug in Atom Dock that's throwing a lot of stuff). So to make some sense of the log try to do it with just one Extension at a ime.
